File: app/src/main/java/com/example/ricardo/tickit2/data/entity/Ticket.java
package com.example.ricardo.tickit2.data.entity;

/**
 * Created by igulu on 13/11/2017.
 */

public class Ticket {
    private Long id;
    private Long parentTicketTypeID;
    private float actualPrice;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getParentTicketTypeID() {
        return parentTicketTypeID;
    }

    public void setParentTicketTypeID(Long parentTicketTypeID) {
        this.parentTicketTypeID = parentTicketTypeID;
    }

    public float getActualPrice() {
        return actualPrice;
    }

    public void setActualPrice(float actualPrice) {
        this.actualPrice = actualPrice;
    }
}

File: app/src/main/java/com/example/ricardo/tickit2/data/model/Ticket.kt
package com.example.ricardo.tickit2.data.model

import android.annotation.SuppressLint
import android.os.Parcelable
import com.example.ricardo.tickit2.data.dto.OrderDto
import com.example.ricardo.tickit2.data.dto.TicketDto
import kotlinx.android.parcel.Parcelize

/**
 * Created by Ricardo on 2018/1/3.
 */
@SuppressLint("ParceCreator")
@Parcelize
class Ticket(
        val id: String,
        val studentID: String,
        val actualPrice: Int,
        val status: Int,
        val createTime: String,
        val ticketTypeID: String,
        val showName: String,
        val showAvatar: String,
        val showDescription: String

): Parcelable {
    constructor(dto: TicketDto):this(
            id = dto.pk,
            studentID = dto.fields.studentID,
            actualPrice = dto.fields.actualPrice,
            status = dto.fields.orderStatus,
            createTime = dto.fields.createTime,
            ticketTypeID = dto.fields.ticketTypeID,
            showName = dto.fields.showName,
            showAvatar = dto.fields.showAvatar,
            showDescription = dto.fields.showDescription
    )
}